tools/ci-cdk/canary-lambda/src/latest/s3_canary.rs

```diff
@@ -207,10 +408,23 @@ async fn test_s3_canary() {
     let config = aws_config::load_from_env().await;
     let client = s3::Client::new(&config);
     s3_canary(
-        client,
+        client.clone(),
         std::env::var("TEST_S3_BUCKET").expect("TEST_S3_BUCKET must be set"),
+    )
+    .await
+    .expect("success");
+
+    s3_mrap_canary(
```

Comment:
Consider `tokio::spawn`ing these futures and awaiting all of them at the bottom so that progress can be made on one while another is waiting, unless they interfere with each other.

Reply:
Addressed in 5ac4b4a